### ISSUE TITLE
Hi can we simply juste use a gsm modem and gsmevil2 for getting the number ?

### ISSUE THREAD

**Issue by Kalvelign** [theapache64/sim-jacker#7]
Can we just simply go to like a restorant and use the arround phone for send and resive sms with the imsi number ? and like use that for sms fraud , if yes can you help figure it out ?

**Comment by joldboy69** [comment]
I have tried that (under a controlled environment) but its been a while and as far as i remember, setting up the correct drivers can be a pain so don't be surprised if you get no output

**Comment by Kalvelign** [comment]
can you provide any source for helping me doing that mayby i can set up it and make it work

**Comment by joldboy69** [comment]
sure. just a few questions. do you have a hackrf? also, maybe try https://github.com/ninjhacks/gsmevil2 for setting up gsmevil2. i used kali linux to do it on vmware

**Comment by Kalvelign** [comment]
i dont have an hackrf but i use my phone for capturing imsi

**Comment by joldboy69** [comment]
may i ask how? is there a project you have for doing this?

**Comment by Kalvelign** [comment]
ok thx you for the general idée and the project im working on if we can call that a project but the general idée is for showing the risk of cell tower and cell phone and all the risk we take by trusting the cellular network , im just gonna look more into it and try to get sms from the other imsi around me and mayby right a little pdf for my scool for showing all the risk and mayby skip some years in scool.

**Comment by joldboy69** [comment]
Okay. Just make sure you have permission to do that. Do you mind sharing a snippet of your code so far? You said you were doing this with a mobile phone (i assume it runs android). Is it rooted?

**Comment by Kalvelign** [comment]
yes im using a rooted android with gsmevil2 and is pretty hard to run it on phone but its possible and for the gsm antena im using an usb antean with adaptator or the antena from my phone ( samsung galaxie 9 plus )

**Comment by joldboy69** [comment]
Wow. Im impressed that actually works. So i assume you were able to succesfully capture imsi numbers? (If i follow correctly)

**Comment by Kalvelign** [comment]
Yea , but you can lake an imsi cacher with pretty mush any antena

**Comment by joldboy69** [comment]
Well done. So I assume, as i stated above, the only problem you seem to face is actually extracting text messages using those imsi numbers. Maybe you can try reverse engineering the Google Messages app to change your imsi number and cell phone number in the app and see what happens (for chat features over wifi). To do that you can try using https://hack-app-data.fileplanet.com/apk.
Can't guarantee that its 100 percent malware free but it works great with rooted devices.

**Comment by Kalvelign** [comment]
yea the only probleme il facing is taht recive and send sms with the imsi number

**Comment by joldboy69** [comment]
hm. part of the problem is that you and i both dont have the fancy expensive  "ideal" equipment to do this. try using google messages and using hack app data to edit your imsi number and phone number, if you haven't already. i cant guarantee that will work but its worth a try. you will also need to figure out the phone number linked to that imsi as well as the carrier (https://www.freecarrierlookup.com/). imei probably wont be required but i cant be sure.

**Comment by Kalvelign** [comment]
ok thx for the link i will try and most my result
